README.md

```diff
@@ -56,7 +56,8 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
 * [express-boilerplate](https://github.com/giladno/express-boilerplate) - A simple Node.js app (Express + Sequelize) with the purpose of providing a compact, clear and concise example to how Node.js apps can be built
 * [mocha-sidebar](https://github.com/maty21/mocha-sidebar) - Mocha side bar is a visual studio code extension that allows you to run Mocha tests from side bar menu and view results can run each level hierarchy from all tests to a single test(and each describer of course)
 * [mean.io](https://github.com/linnovate/mean) - Mean.io is a popular mean stack boilerplate providing a starting point for angular(6) , express, mongo and node fullstack javascript powered apps.
-
+* [SugoiJS](https://github.com/sugoiJS) - SugoiJS provides TypeScript decorators which makes the development process faster, easier with less boilerplate code.
```

Comment:
Orel this is the organization address. Did you mean to list and point to https://github.com/sugoiJS/core ?

Reply:
It's actually contains 4 packages core,server,socket and orm
What is the best way to represent it?

Reply:
If you really don't have one highlighted project out of the whole organization I'll go ahead and merge it.